testing/smoke-ide-test/src/smokeIdeTest/groovy/org/gradle/ide/sync/IsolatedProjectsParallelSyncTest.groovy

```diff
@@ -29,16 +28,11 @@ class IsolatedProjectsParallelSyncTest extends AbstractIdeaSyncTest {
         server.start()
     }
 
-    @ToBeImplemented
     def 'projects are configured in parallel during IDEA sync'() {
         given:
         simpleProject()
         server.expect("configure-root")
-        server.expect("configure-a")
-        server.expect("configure-b")
-        // TODO: isolated expected behaviour for parallel sync
-//        server.expect("configure-root")
-//        server.expectConcurrent("configure-a", "configure-b")
+        server.expectConcurrent("configure-a", "configure-b")
```

Comment:
This test actually works like this even on `master` without changes in this PR. I am able to reproduce it locally.

The reason for this is related to `BuildType.TASKS` not meaning "only run tasks, no models are built". Instead it means "tasks are requested, but models might as well". Another part of the story is that any IDE sync (IJ 2024.2 and IJ 2024.2.1) is identified as `BuildType.TASKS` even if no tasks are involved. Instead, I observe a request for "default tasks".